neon_mq_connector/connector.py

```diff
@@ -74,6 +75,7 @@ def __init__(self, config: dict, service_name: str):
             :param service_name: name of current service
        """
         self.config = config
+        self.service_id = self.create_service_id()
```

Comment:
This should be an internal param `self._service_id` with an optional `@property` method if an external service might need to get this